runner/execer/os/os_execer.go

```diff
@@ -229,7 +231,7 @@ func (e *osExecer) monitorMem(p *osProcess, memCh chan execer.ProcessStatus) {
 						"tag":    p.Tag,
 						"jobID":  p.JobID,
 						"taskID": p.TaskID,
-					}).Debugf("ps after increasing mem_cap utilization for pid %d", pid)
+					}).Tracef("ps after increasing mem_cap utilization for pid %d", pid)
```

Comment:
This log is super verbose so I get dropping it to Trace, but it's come in handy a few times when profiling high mem usage tasks

Reply:
yes, although when we finally switch staging to prod, we'll lose debug level anyway. with using the os execer for fs_util runs, the log really becomes unusable, with multiple ps outputs per task in remexec